Dockerfile.docs

```diff
@@ -7,9 +7,17 @@
 # Useful for local testing
 
 FROM python:3-slim
-RUN apt-get update && apt-get install --no-install-recommends -y git \
-  && pip install mkdocs "mkdocs-material>=8.2" "mkdocs-htmlproofer-plugin>=0.8" "mkdocs-swagger-ui-tag" && \
+RUN apt-get update && apt-get install --no-install-recommends -y git wget unzip \
+  && pip install mkdocs "mkdocs-material>=8.2" "mkdocs-htmlproofer-plugin>=0.8" \
+  "mkdocs-swagger-ui-tag" "mkdocstrings>=0.24.3" "mkdocstrings-python>=1.9.2" && \
   rm -rf /var/lib/apt/lists/*
+
+WORKDIR /apis
+RUN wget https://github.com/ejlee3/performance-tools/archive/refs/heads/add-sphinx.zip
```

Comment:
yes - I need to merge the other PR first, then we can do this, but I left it this way for now so that people can try it out and make sure it works for them too.